docs/source/serving/compatibility_matrix.rst

```diff
@@ -393,7 +393,7 @@ Feature x Hardware
      - ✅
      - ✅
      - ✅
-     - ✗
+     - ?
```

Comment:
did you intend to include this?  just checking since it doesn't look related to the PR title and wasn't mentioned in the summary

Reply:
Yes, let me update the title.